bin/main.rb

```diff
@@ -1,73 +1,51 @@
-class Player
-  attr_accessor :name, :weapon
+require '../lib/player.rb'
+require '../lib/board.rb'
+require '../lib/game.rb'
```

Comment:
Use require_relative here, I couldn't run this file

Reply:
do you mean this?
```
require_relative '../lib/player.rb'
require_relative '../lib/board.rb'
require_relative '../lib/game.rb'
```

or do you mean this?
```
require_relative 'player.rb'
require_relative 'board.rb'
require_relative 'game.rb'
```

Reply:
Sorry but please change it to `require_relative '../lib/player.rb`
I've got this error when trying to run it
```ruby
$ ruby bin/main.rb 
Traceback (most recent call last):
	2: from bin/main.rb:1:in `<main>'
	1: from /home/elvisfosso/.rbenv/versions/2.5.1/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/home/elvisfosso/.rbenv/versions/2.5.1/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- ../lib/player.rb (LoadError)
```

I ran it after changing `require` to `require_relative`